### PR TITLE
Challenge 8, added slugs.js and test spec for W&C news title

### PR DESCRIPTION
**Pull request by milicazivanovic-work**

### CODE REVIEW

File: cypress/support/slugs.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+var news_slugs = [
```
Comment (srdjadzogaz) [inline cypress/support/slugs.js:1]: This is good example of what fixture should be. Move this test data into fixtures and update import function in the e2e test.

File: cypress.saucedemo.config.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ module.exports = defineConfig({
   chromeWebSecurity: false,
   e2e: {
     baseUrl: 'https://www.saucedemo.com/',
-    excludeSpecPattern: ['cypress/e2e/homepage.cy.js', 'cypress/e2e/news.cy.js', 'cypress/e2e/spec.cy.js'],
+    excludeSpecPattern: ['cypress/e2e/homepage.cy.js', 'cypress/e2e/news.cy.js', 'cypress/e2e/spec.cy.js', 'cypress/e2e/check-news-pages.cy.js'],
```
Comment (srdjadzogaz) [inline cypress.saucedemo.config.js:7]: Just a note you can use here spec pattern for example * or ** to capture nested folders. also you could create naming schema and capture it in the pattern mobile, prod, test, env etc. Just a note no need to edit anything.

**Review by srdjadzogaz — APPROVED** [review]

**Comment by srdjadzogaz** [comment]
Bravo :) very good

File: cypress/support/slugs.js (hunk 1 above)
Comment (milicazivanovic-work) [reply cypress/support/slugs.js]: Thank you, done! :)

**Pull request merged**